scout/supporting_data/config_schema.yml

```diff
@@ -0,0 +1,243 @@
+        type: object
+        properties:
+          exog_hp_rate_scenario:
+            type: ["string", "null"]
+            enum: [conservative, optimistic, aggressive, most aggressive, null]
+            description: Guidehouse E3HP conversion scenario to use for exogenous HP rates.
+          switch_all_retrofit_hp:
+            type: boolean
+            description: Assume all retrofits convert to heat pumps, otherwise, retrofits are subject to the same external heat pump conversion rates assumed for new/replacement decisions. Only applicable if `exog_hp_rate_scenario` and `retrofit_type` are defined.
+        if:
+          properties:
+            exog_hp_rate_scenario:
+              enum: [conservative, optimistic, aggressive, most aggressive]
+        then:
+          required: [exog_hp_rate_scenario]
+
+      alt_ref_carb:
+        type: boolean
+        default: false
+        description: set the baseline electricity emissions intensities to be consistent with the Standard Scenarios Mid Case (rather than AEO).
+
+      grid_decarb:
+        type: object
+        required: [grid_decarb_level, grid_assesment_timing]
+        properties:
+          grid_decarb_level:
+            type: ["string", "null"]
+            enum: ["0.8", full, null]
+            description: Enable grid decarbonization - `0.8` represents 80%% reduced grid emissions from current levels by 2050, and `full` represents full grid decarbonization by 2035.
+          grid_assesment_timing:
+            type: ["string", "null"]
+            enum: [before, after, null]
+            description: When to assess avoided emissions and costs from non-fuel switching measures relative to the additional grid decarbonization.
+
+      adopt_scn_restrict:
+        type: ["string", "null"]
+        enum: [Max adoption potential, Technical potential, null]
+        description: Specify adoption scenario restrictions.
+
+      retrofits:
+        type: object
+        required: [retrofit_type]
+        properties:
+          retrofit_type:
+            type: ["string", "null"]
+            enum: [constant, increasing, null]
+            description: Type of retrofit. `constant` assumes component-based early retrofit rates that do not change over time, `increasing` assumes rates that increase over time.
+          retrofit_multiplier:
+            type: ["number", "null"]
+            description: Factor by which early retrofit rates are multiplied (only for increasing retrofit_type).
+          retrofit_mult_year:
+            type: ["integer", "null"]
+            description: The year by which the retorfit multiplier is achieved (only for increasing retrofit_type).
+        if:
+          properties:
+            retrofit_type:
+              const: increasing
+        then:
+          required: [retrofit_multiplier, retrofit_mult_year]
+
+      add_typ_eff:
+        type: boolean
+        default: false
+        description: Enable additional typical efficiency.
```

Comment:
Rephrase to something like "Enable automatic addition of ECMs with typical (business-as-usual) efficiency to compete with user-defined ECMs."